### DISCUSSION TITLE
Doc Tools for C++ Libraries

### DISCUSSION THREAD

**Discussion by tronical** [slint-ui/slint#295] [Blog Comments]
This is the discussion thread for the blog post [Doc Tools for C++ Libraries](https://sixtyfps.io/blog/documentation-tools-for-cpp-libraries.html).

**Comment by bryceschober** [comment]
@tronical Your post didn't say much about why you chose Sphinx + etc. vs. Standardese... In thinking about our starting a new project with documentation, what were your trade-offs?

**Reply by tronical** [reply]
At the time of choosing I wasn’t quite so aware of standardese. I don’t quite like the mixture of many components (Sphinx, breathe, exhale, doxygen, myst-parser, etc) but otoh I love the read the docs theme. Look and feel goes a long way :). Cross referencing is one of the features I find super important, it needs to be easy to make links from conceptual docs to api and vice versa. It’s not easy but so far I think we’ve always found a way.

Happy to help of you have specific questions:)
